spec/language/Ch04_Testbench_Templates.md

```
@@ -0,0 +1,807 @@
+
+```scriban
+{{ for supply in harness.supplies }}
+V{{ supply.net }} {{ supply.net }} 0 DC {{ supply.value }}
+{{ end }}
+```
+
+- `supply.net`: net name (e.g., `"VDD"`)
+- `supply.value`: voltage value (e.g., `"1.8V"`)
+
+**harness.loads** (array of objects):
+```scriban
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)